index.js

```diff
@@ -45,7 +45,11 @@ module.exports = function accessToken(options = {}) {
 
   if (opts.expireMode === expireModes.at) {
     const expirationDateValue = getExpirationDate(this, opts.expired);
-    const expirationDate = opts.parseDate ? expirationDateValue : expirationDateValue.toISOString();
+    const expirationDate = {
```

Comment:
Could we only compute the method we are looking for instead of computing every possible value? Perhaps a getter or function would make the work.

Reply:
Why do you want to optimize this? This is testing code and in my initial implementation I wanted to favor readability and shortness over avoiding unnecessary computations. After all I assume that the additional "computation" is really negligible runtime-wise.

Nonetheless I proposed a change that uses a switch statement. I personally still prefer the shortness of the first solution but I'll leave it up to you to pick whatever you feel fits this library.

Reply:
Got it, the new proposal sounds good. Even better that it doesn't break when the configuration option is invalid. Could we also update the documentation?

Reply:
I'm sorry I missed the documentation. Just added a commit.